Entity/SelectedStopPointRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace CanalTP\MttBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+/**
+ * SelectedStopPointRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class SelectedStopPointRepository extends EntityRepository
```

Comment:
Yes I know because I think the entities were generated with the command lin doctrine:entites:...
that generate empty repositories.

You can keep them. don't do extra work it's ok